react/components/ItemTitle.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+import { Link } from 'vtex.render-runtime'
+
+import categoryMenuPosition, {
+  getMenuPositionValues,
+} from '../utils/categoryMenuPosition'
+
+import { itemPropType } from '../propTypes'
+
+const ItemTitle = ({ item: { name, slug, params, page }, showBorder, onClick, menuPosition }) => {
+  const classes = classNames(
+    'w-100 pv5 no-underline t-small outline-0 db tc ttu link truncate bb bw1 c-muted-1', {
+      'b--transparent': !showBorder,
+      'b--action-primary pointer': showBorder,
+      'mr8': menuPosition === categoryMenuPosition.DISPLAY_LEFT.value,
+      'ml8': menuPosition === categoryMenuPosition.DISPLAY_RIGHT.value,
+      'mh6': menuPosition === categoryMenuPosition.DISPLAY_CENTER.value,
+    }
+  )
+
+  return (slug || (page && params)) ? (
+    <Link
+      onClick={onClick}
+      to={slug}
+      page={page}
+      params={params}
+      className={classes}
+    >
+      {name.toUpperCase()}
```

Comment:
I think we could change this behaviour of changing to uppercase and only follow the typography tokens.

I'm having an issue in Ecowater that I can't make the menu like they want because of that change in case.

cc @gabrielgalc

Reply:
You are right, Breno! The design tokens should be the only responsible for typography and color selection